src/Stratis.Bitcoin.Features.IndexStore/IndexStoreFeature.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.BlockPulling;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Builder.Feature;
+using Stratis.Bitcoin.Configuration;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Features.BlockStore;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.Features.RPC.Controllers;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.IndexStore
+{
+    public class IndexStoreFeature : BlockStoreFeature
```

Comment:
`IndexStoreFeature` was deleted from this repo and moved to the BlockExplorer repo, I think you didn't merge correctly.